src/collections/_documentation/common/getting_started/getting_started_template.md

```diff
@@ -0,0 +1,124 @@
+<!--
+Guideline: This page is common to all SDKs; add the name of the SDK you're documenting as well as the code samples as instructed. If you have questions, please ask Fiona or Daniel. 
+
+**The objective for this page is that a developer can quickly complete an ideal integration.**
+-->
+
+Add Sentry's <!--SDK name--> SDK to automatically report errors, exceptions, and rejections in your application. 
+
+On this page, we provide concise information to get you up and running quickly. To learn more, use the links to in-depth content.
+
+In the right place? We also offer documentation for:
+<!-- 
+Guideline: Add icons for related languages/frameworks.
+-->
+What's covered on this page:
+
+- [Install](#install)
+- [Initialize](#initialize)
```

Comment:
nit: should we call this configure?

Reply:
Since it's an `init` call, I thought to leave it as initialized. We have the ability to configure your SDK in terms of turning on/off debug mode, setting a sampleRate, defining maxBreadcrumbs and such in the content dedicated to configure. If we change this to configure here, will that confuse the readers?

Reply:
you pass configuration all in the same place... init is just the name of the call at the end of the day. that's why I would call it configure

Reply:
Done.